CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 - ([#269](https://github.com/demos-europe/demosplan-ui/pull/269)) DpIcon: new selector `landscape`, `portrait`, or `square`, based on icon proportions ([@spiess-demos](https://github.com/spiess-demos))
 - ([#270](https://github.com/demos-europe/demosplan-ui/pull/270)) DpIcon: new icons (ai, copy) ([@spiess-demos](https://github.com/spiess-demos))
+- ([#268](https://github.com/demos-europe/demosplan-ui/pull/268)) Tests: Add Testsetup for Vue Files and some Tests ([@salis-demos](https://github.com/salis-demos))
```

Comment:
don't we add the newest entry at the top of the section?

Reply:
do we? We can though. I haven't thought about it.

Reply:
changed in 3edbde43e14a994be1aa5c2bca4282688c7b255c